middlewares/cardCheckPermission.js

```diff
@@ -0,0 +1,17 @@
+const Card = require('../models/card');
+const NotFoundError = require('../errors/notFoundError');
+
+module.exports = (req, res, next) => {
+  const { cardId } = req.params;
+
+  Card.findById(cardId)
+    .orFail(() => new NotFoundError('Карточка не найдена'))
+    .then((card) => {
+      if (!card.owner.equals(req.user._id)) {
+        res.status(403).send({ message: 'Недостаточно прав' });
+        return;
+      }
```

Comment:
Можно лучше: создать отдельный класс ошибки для 403.
По аналогии с 401 и 404 можно создать отдельный класс и бросать исключение. Далее всю необходимую обработку сделает `.catch()` блок:
```suggestion
        return Promise.reject(new ForbiddenError('Недостаточно прав'));
```
либо `throw`:
```suggestion
        throw new ForbiddenError('Недостаточно прав');
```